atom/common/native_mate_converters/net_converter.cc

```diff
@@ -191,16 +191,34 @@ bool Converter<net::HttpResponseHeaders*>::FromV8(
     if (!keys->Get(i)->ToString(context).ToLocal(&key)) {
       return false;
     }
-    if (!headers->Get(key)->ToString(context).ToLocal(&value)) {
-      return false;
+
+    if (headers->Get(key)->IsArray()) {
+      auto values = v8::Local<v8::Array>::Cast(headers->Get(key));
+      for (uint32_t j = 0; j < values->Length(); j++) {
+        if (!values->Get(j)->ToString(context).ToLocal(&value)) {
+          return false;
+        }
+
+        v8::String::Utf8Value key_utf8(key);
```

Comment:
This appears to be duplicated work (from line 214) and I feel as though we should be able to use a `mate::ConvertFromV8` call here instead of the `ostringstream` logic.